chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -390,10 +433,13 @@ impl Actor for PeerManagerActor {
                     state.config.event_sink.push(Event::ServerStarted);
                     loop {
                         if let Ok((conn, client_addr)) = listener.accept().await {
-                            peer_manager_addr.do_send(PeerToManagerMsg::InboundTcpConnect(
-                                InboundTcpConnect::new(conn),
-                            ));
+                            // Always let the new peer to send a handshake message.
```

Comment:
In theory, we can wrap this whole `ctx.spawn` in `if self.config.is_inbound_disabeld`. out of scope for PR though.